imports/ui/pages/upload/ipfs-uploader.js

```diff
@@ -28,6 +28,51 @@ document.addEventListener('DOMContentLoaded', (e) => {
   }
 })
 
+function renderForm (formEl, videoObject, cb) {
+  // formEl.innerHTML = '<form class="upload-form" method="POST" action="#">'
+  formEl.innerHTML += 'Video Title:'
+  formEl.innerHTML += '<input name="title" id="input-title" type="text" class="form-control textbox">'
+  formEl.innerHTML += '<h5>screenshots</h5><ul>'
+
+  let screenshots = videoObject.screenshots
+  for (let i = 0; i < screenshots.length; i++) {
+    formEl.innerHTML += '<li><input type="radio" name="screenshot" value="https://gateway.paratii.video/ipfs/' + videoObject.master.hash + '/' + screenshots[i] + '">'
+    formEl.innerHTML += '<img width="240" src="https://gateway.paratii.video/ipfs/' + videoObject.master.hash + '/' + screenshots[i] + '"></li>'
+  }
+
+  formEl.innerHTML += '</ul>'
+  formEl.innerHTML += '<br><input class="btn btn-default" type="submit" value="submit">'
```

Comment:
Just a small note: I would think we might want to avoid assigning `innerHTML` so many times and instead construct a string of the html piece by piece and at the end assign it to `formEl.innerHTML`. Reason being so that the string parsing/conversion to DOM nodes only happens once, rather than in increasingly large pieces multiple times.

I doubt this would have a huge performance impact but just some low-hanging fruit.